<?xml version="1.0"?>
<launch>
    <rosparam file="$(find angle_aware_pytorch)/config/yolov5.yaml" command="load" ns="" />
    <rosparam file="$(find angle_aware_switch)/config/central_sky.yaml" command="load" ns="" />
    <node pkg="angle_aware_pytorch" type="object_detector_yolov5.py" name="object_detector_yolov5" output="screen" respawn="false" respawn_delay="0">
        <param name="output_pose_topic" value="grape_posestamped" />
        <param name="pytorch_hub_path" value="$(find angle_aware_pytorch)/yolov5" />
        <param name="model_path" value="$(find angle_aware_pytorch)/model/test1_yolov5s.pt" />
        <rosparam file="$(find bebop_driver)/data/bebop2_camera_calib.yaml" command="load" ns="" />
    </node>
</launch>